nipyapi/utils.py

```diff
@@ -530,11 +530,16 @@ def check_version(base, comparator=None, service='nifi'):
     return 0
 
 
-def validate_parameters_versioning_support():
-    """Convenience method to check if Parameters are supported"""
-    nifi_check = enforce_min_ver('1.10', bool_response=True)
+def validate_parameters_versioning_support(verify_nifi=True, verify_registry=True):
+    """
+    Convenience method to check if Parameters are supported
+    Args:
+        verify_nifi (bool): If True, check NiFi meets the min version
+        verify_registry (bool): If True, check Registry meets the min version
+    """
```

Comment:
can you change the if else structure to match the rest of the code?
if X 
   do
else

Reply:
👍